.github/actions/gradle-setup/action.yml

```diff
@@ -9,7 +9,7 @@ runs:
       with:
         repository: eclipse-dataspaceconnector/DataSpaceConnector
         path: DataSpaceConnector
-        ref: 5191d3e6dd9ac5d78264d05ae69edb4d297b606a
+        ref: fb21f13366979e2e909162b659c2308602985d88
```

Comment:
reminder to change commit when [PR](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1702) is merged